libraries/botbuilder-lg/tests/lg.test.js

```diff
@@ -699,6 +699,9 @@ describe('LG', function() {
         var evaled = templates.evaluateText('hello');
         assert.strictEqual('hello', evaled);
 
+        evaled = templates.evaluateText("${''}");
```

Comment:
evaled = templates.evaluateText("${``}"); 
should evalute an empty string interpolation

Reply:
fixed